Products/LDAPUserFolder/LDAPDelegate.py

```diff
@@ -568,7 +565,7 @@ def _clean_rdn(self, rdn):
         try:
             key, val = rdn.split('=')
             val = val.lstrip()
-            return '%s=%s' % (key, escape_dn_chars(val))
+            return '%s=%s' % (key, val)
```

Comment:
`escape_dn_chars` was called for a reason. Why did you take it out?

Reply:
Actually I don't know:  maybe it was (mis-) interpreted as a py2 residual and is was removed as a exclusion strategy (remove everyting what might cause errors)  - as a first step:
https://github.com/zms-publishing/Products.LDAPUserFolder/commit/9c97e82e6b5905e6bef311aee11a303049bda2e8#diff-f896aace4e37b3a85794c600852fa14d

I added the function again
https://github.com/zms-publishing/Products.LDAPUserFolder/commit/2f6f78add7212b360be65de3ca60bbfd806da491